romancal/datamodels/schemas/reference_files/gain.schema.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
+---
+$schema: http://stsci.edu/schemas/asdf/asdf-schema-1.0.0
+id: "http://stsci.edu/schemas/roman_datamodel/reference_files/gain.schema"
+allOf:
+- $ref: referencefile.schema
+- $ref: keyword_gainfact.schema
+- type: object
+  properties:
+    data:
+      title: The gain
```

Comment:
A minor nit, can we change "The gain" --> "detector gain map"  or something similar.

Reply:
Fixed.